src/webviews/apps/settings/settings.html

```diff
@@ -63,9 +63,9 @@ <h2 class="hero__title">Settings</h2>
 			<div class="content__area content__area--full-scroll">
 				<!-- prettier-ignore -->
 				<%= require('html-loader?{"esModule":false}!./partials/current-line.html') %>
-				<%= require('html-loader?{"esModule":false}!./partials/code-lens.html') %>
+				<!-- <!-- <%= require('html-loader?{"esModule":false}!./partials/code-lens.html') %> -->
```

Comment:
Not sure what is going on with the comments here, but looks like these should be reverted